src/apps/common/submodules/portListing/portListing.js

```diff
@@ -312,7 +312,11 @@ define(function(require) {
 							_.get(portRequest, 'numbers')
 						),
 						unactionableStatuses = ['canceled', 'completed'],
-						isAgent = self.portListingUtilIsAgent(_.get(portRequest, 'port_authority')),
+						isAgent = self.portListingUtilIsAgent(_.get(
+							portRequest,
+							'port_authority',
+							_.get(portRequest, '_read_only.port_authority'))
+						),
```

Comment:
Wouldn't it make sense to check the read only value directly then? Or is in not exposed at all times?

Reply:
Right, it looks it is not including that `port_authority` key at the top of the json object anymore https://github.com/2600hz/kazoo/blob/master/core/kazoo_numbers/src/knm_port_request.erl#L100-L108

Reply:
In that case, could you update your change to simply:
```js
isAgent = self.portListingUtilIsAgent(_.get(portRequest, '_read_only.port_authority')),
```

Reply:
Done, thank you for reviewing 👍